### PR TITLE
Readme copy

### PR DESCRIPTION
**Pull request by digitalbuddha**

### CODE REVIEW

File: README.md
Comment (kenyee) [inline README.md:6]: typo:
"bit" = "big"

Comment (kenyee) [reply README.md:6]: typo "at same time" should be "at the same time"

"with same patterns" should be "with the same patterns"

File: README.md
Comment (kenyee) [inline README.md:8]: "." should be after the link the the thread.

File: README.md
Comment (kenyee) [inline README.md:10]: "Similar to other static check libraries..." should be a new sentence

File: README.md
Comment (kenyee) [inline README.md:16]: "list of rules will check for" should be "list of rules"

**Comment by digitalbuddha** [comment]
recheck

File: README.md
Comment (mrmans0n) [inline README.md:8]: That rogue dot probably has to go "...to be aware of[.] as seen in this..."

File: README.md
Comment (kenyee) [inline README.md:46]: should replace "," with ";"

**Review by kenyee — CHANGES_REQUESTED** [review]

File: README.md
Comment (digitalbuddha) [reply README.md]: just caught the bit one, on the other one, btw you can do it as a suggestion so I can auto apply

**Review by mrmans0n — APPROVED** [review]
Remove the dot and good to go

File: README.md
Comment (kenyee) [reply README.md:8]: " aware of. [as " should be " aware of, [as "

**Comment by digitalbuddha** [comment]
I think that's all the comments, I can't merge until approved or maybe I'm not fancy enough to merge at all

**Comment by mrmans0n** [comment]
You'll need to say the "I have read the CLA Document and I hereby sign the CLA" btw, only signing the CLA without that didn't work for me (I tried rechecking and still fails your CLA compliance)

**Comment by digitalbuddha** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by digitalbuddha** [comment]
hmm still nothing, going to go for a walk and try again  and go to sleep Nacho!

**Comment by mrmans0n** [comment]
I've forced the recheck and it should pass now.

**Review by kenyee — APPROVED** [review]

**Pull request merged**
